src/libsystemd-network/fuzz-dhcp-server.c

```diff
@@ -62,9 +65,11 @@ static int add_static_lease(sd_dhcp_server *server, uint8_t i) {
 }
 
 int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+        _cleanup_(rmdir_and_freep) char *tmpdir = NULL;
```

Comment:
I think it should be `rm_rf_physical_and_freep` instead. With `rmdir_and_freep` the fuzz target doesn't remove temporary lease files and fills up `/tmp`.